src/open_r1/configs.py

```diff
@@ -14,11 +14,108 @@
 # limitations under the License.
 
 from dataclasses import dataclass, field
-from typing import Optional
+from typing import Any, Dict, List, Optional
```

Comment:
Now we should use `list` and `dict`

Reply:
Done in https://github.com/huggingface/open-r1/pull/647/commits/ba3a68760a48a4302abf91b8fa8d5a3f9cc4f74a